week03/html/home.html

```diff
@@ -0,0 +1,51 @@
+<link href="../css/home.css" rel="stylesheet" /> 
+<link rel="icon" type="image/png" sizes="16x16" href="../src/billyLogo.png"/>
+
+<title>Lee Gabin</title>
+
+<body>
+
+  <!-- 상단바 -->
+
+  <header id="header">
+  <div class="navbar">
+    <div class="nav1">
+      <a href="../html/home.html" class="logo">
+        <img src="../src/mainLogo.png" height="70px" />
+      </a>
+    </div>
+    <div class="nav2"></div>
+    <div class="nav3">
+      <ul>
+        <li><a href="../html/home.html" style="color: black;">Home</a></li>
+        <li><a href="../html/about.html">About</a></li>
+        <li><a href="../html/project.html">Project</a></li>
+        <li><a href="../html/activity.html">Activity</a></li>
+        <li><a href="../html/contact.html">Contact</a></li>
+      </ul>
+    </div>
+  </div>
+  </header>
+
+  <!-- 메인 내용 -->
+
+  <div class="intro">
+    <!-- <div style="float: left"> -->
+    <img src="../src/HomePic.png" />
+    <div class="content">
+    <div class="hello">This is <a id="job">&nbsp;designing developer</a> </div>

```

Comment:
1. 이런 경우, `a` 태그보다 `span` 태그를 사용하는 것이 좋습니다.
   - `a` 태그는 주로 링크를 통해 다른 메뉴나 다른 웹사이트로 이동하는 용도로 쓰입니다.
   - `span` 태그는 텍스트(문자) 등을 꾸미기 위한 용도로 쓰입니다.

2. `&nbsp;`를 사용할 필요가 없습니다.
   - `&nbsp;`는 “Non-breaking Space”를 의미하는 HTML entity로, `&nbsp;`를 사용하면 띄어쓰기는 되지만 자동으로 줄바꿈되지 않습니다.
   - `a` 태그 또는 `span` 태그는 줄바꿈 속성이 없는 inline 요소이므로 `&nbsp;`를 사용할 필요가 없습니다.

```suggestion
    <div class="hello">This is <span>designing developer</span></div>
```